### ISSUE TITLE
Workflow is now failing

### ISSUE THREAD

**Issue by gvallangca317** [pratikmallya/autolabeler-codeowners#27]
Failure provides 1 error and 2 warnings

<img width="1516" alt="image" src="https://github.com/pratikmallya/autolabeler-codeowners/assets/87317431/80e47918-9dcd-428e-ab23-ba23afaa024d">

**Comment by rosswaycaster** [comment]
My assumption is that this error originates from the block where the label is being created: https://github.com/pratikmallya/autolabeler-codeowners/blob/master/src/applyLabels.ts#L15-L33.

I suspect it is attempting to recreate labels that already exist, leading to an error being thrown.

**Comment by gvallangca317** [comment]
> My assumption is that this error originates from the block where the label is being created: https://github.com/pratikmallya/autolabeler-codeowners/blob/master/src/applyLabels.ts#L15-L33.
> 
> I suspect it is attempting to recreate labels that already exist, leading to an error being thrown.

Is that an issue that can be fixed on the PR or something should be done on the codeblock? I'm assuming GH did something new that is causing to break on that codeblock

**Comment by rosswaycaster** [comment]
Yeah it looks like GitHub changed their error message - I think the catch block might need to be updated:

```tsx
// if label already exists
if (error.code !== 'already_exists') {
  throw error
}
```

**Comment by gvallangca317** [comment]
Do you know if GH had any documentation on the error messaging change?
